plugins/woocommerce/client/blocks/assets/js/base/stores/cart-items.ts

```diff
@@ -149,17 +135,17 @@ export const { state, actions } = store< Store >(
 
 				try {
 					const res: Response = yield fetch(
-						`${ state.restUrl }wc/store/v1/cart/items`,
+						`${ state.restUrl }wc/store/v1/cart`,
```

Comment:
I think we'll need to update the e2e tests to change which endpoint they're waiting to resolve and that's possibly why quite a few are failing right now.

Reply:
Ok, I'll look into it. Thanks.